### PR TITLE
docs: add windows command to all sample

### LINKED ISSUE #13: Multi-Platform Sample Instructions
EDC samples contain a README file with step by step instructions on how to run them on Linux/Mac. Developers using Windows need to adapt the commands (e.g. curl commands) if they want to run the samples e.g. using Powershell.

To make it easier for a broader range of developers to run the samples to get started, it would be beneficial if the instructions would contain instructions for both Linux/Mac and Windows.

This issue proposes to define and implement the following requirements for samples:

- Samples should be able to run natively on Mac/Linux and Windows
  - Tools like the Windows Subsystem for Linux (WSL) should not be required when running a sample on Windows
  - Additional tooling required to run a sample should be suited to the skill level at which the sample is aimed. It is OK for certain samples to use Docker to avoid installation of additional tools like e.g. Jaeger etc.
- A README file with instructions on how to run the sample on both Linux/Mac and Windows should be provided for every sample

### PR DESCRIPTION
**Pull request by mukta2022**
## What this PR changes/adds

This PR change the all sample readme file with windows command

## Why it does that
 
This change was necessary to help for windows/Linux users both

## Linked Issue(s)

Closes #13

## Checklist

- [ ] performed checkstyle check locally?
- [ ] added/updated copyright headers?
- [ ] documented public classes/methods?
- [ ] documented code?
- [ ] assigned appropriate label?
- [ ] formatted title correctly?

### CODE REVIEW

File: transfer/transfer-01-file-transfer/README.md [hunk 1]
```diff
@@ -208,6 +214,9 @@ of the header has to match the value of the `edc.api.auth.key` property in the c
 ```bash
 curl -X POST -H "Content-Type: application/json" -H "X-Api-Key: password" -d @transfer/transfer-01-file-transfer/contractoffer.json "http://localhost:9192/api/v1/management/contractnegotiations"
 ```
+```Windows command Prompt
```
Comment (jimmarino) [inline transfer/transfer-01-file-transfer/README.md:217]: capitalization

File: transfer/transfer-01-file-transfer/README.md [hunk 2]
```diff
@@ -268,7 +277,9 @@ transfer. Then run:
 ```bash
 curl -X POST -H "Content-Type: application/json" -H "X-Api-Key: password" -d @transfer/transfer-01-file-transfer/filetransfer.json "http://localhost:9192/api/v1/management/transferprocess"
 ```
-
+```Windows command prompt
```
Comment (jimmarino) [inline transfer/transfer-01-file-transfer/README.md:280]: capitalization

File: transfer/transfer-02-file-transfer-listener/README.md [hunk 3]
```diff
@@ -52,6 +52,13 @@ java -Dedc.fs.config=transfer/transfer-02-file-transfer-listener/file-transfer-l
 ./gradlew transfer:transfer-01-file-transfer:file-transfer-provider:build
 java -Dedc.fs.config=transfer/transfer-01-file-transfer/file-transfer-provider/config.properties -jar transfer/transfer-01-file-transfer/file-transfer-provider/build/libs/provider.jar
 ````
+```Windows command prompt
```
Comment (jimmarino) [inline transfer/transfer-02-file-transfer-listener/README.md:55]: spacing

File: transfer/transfer-02-file-transfer-listener/README.md [hunk 4]
```diff
@@ -63,6 +70,11 @@ curl -X POST -H "Content-Type: application/json" -H "X-Api-Key: password" -d @tr
 curl -X GET -H 'X-Api-Key: password' "http://localhost:9192/api/v1/management/contractnegotiations/{UUID}"
 curl -X POST -H "Content-Type: application/json" -H "X-Api-Key: password" -d @transfer/transfer-01-file-transfer/filetransfer.json "http://localhost:9192/api/v1/management/transferprocess"
 ```
+```Windows command prompt
```
Comment (jimmarino) [inline transfer/transfer-02-file-transfer-listener/README.md:73]: spacing

**Review by jimmarino — CHANGES_REQUESTED** [review]
Thanks for the contribution. Can you please clean up the capitalization, spacing, etc?

Also, when commands do not change, e.g., curl, please don't duplicate them for the Windows version, as it increases maintenance. Those sections can be removed.

File: basic/basic-01-basic-connector/README.md [hunk 5]
```diff
@@ -25,6 +25,11 @@ with that we can build and run the connector from the root directory:
 java -jar basic/basic-01-basic-connector/build/libs/basic-connector.jar
 ```
 
+```windows command prompt
```
Comment (juliapampus) [inline basic/basic-01-basic-connector/README.md:28]: The type after "```" helps markdown to render the document and is not shown to the user, as you can see:
image

I would therefore suggest the following
```suggestion
```bash
# for Windows users
```

This way, the promt is rendered correctly and readers know that this responds to Windows users.

File: transfer/transfer-01-file-transfer/README.md [hunk 6]
```diff
@@ -176,6 +176,13 @@ java -Dedc.fs.config=transfer/transfer-01-file-transfer/file-transfer-consumer/c
 java -Dedc.fs.config=transfer/transfer-01-file-transfer/file-transfer-provider/config.properties -jar transfer/transfer-01-file-transfer/file-transfer-provider/build/libs/provider.jar
 ````
 
+```windows command prompt
+.\gradlew transfer:transfer-01-file-transfer:file-transfer-consumer:build
+java -Dedc.fs.config=transfer\transfer-01-file-transfer\file-transfer-consumer\config.properties -jar transfer\transfer-01-file-transfer\file-transfer-consumer\build\libs\consumer.jar
+# in another terminal window:
```
Comment (juliapampus) [inline transfer/transfer-01-file-transfer/README.md:182]: Blank line before "# in another terminal"

File: transfer/transfer-05-file-transfer-cloud/README.md [hunk 7]
```diff
@@ -191,6 +202,11 @@ cd transfer/transfer-05-file-transfer-cloud/terraform
 terraform destroy
 ```
 
+```windows command prompt
+cd transfer\transfer-05-file-transfer-cloud\terraform 
+'Second command works for all OS user, So look up!'
```
Comment (juliapampus) [inline transfer/transfer-05-file-transfer-cloud/README.md:207]: Inconsistency with the comments above:

```suggestion
# Second command works for all OS user, so look up!
```

File: transfer/transfer-05-file-transfer-cloud/README.md [hunk 8]
```diff
@@ -94,6 +98,13 @@ java -Dedc.fs.config=transfer/transfer-05-file-transfer-cloud/cloud-transfer-con
 java -Dedc.fs.config=transfer/transfer-05-file-transfer-cloud/cloud-transfer-provider/config.properties -jar transfer/transfer-05-file-transfer-cloud/cloud-transfer-provider/build/libs/provider.jar
 ```
 
+```windows command prompt
+.\gradlew clean build
+java -Dedc.fs.config=transfer\transfer-05-file-transfer-cloud\cloud-transfer-consumer\config.properties -jar transfer\transfer-05-file-transfer-cloud\cloud-transfer-consumer\build\libs\consumer.jar
+# in another terminal window:
```
Comment (juliapampus) [inline transfer/transfer-05-file-transfer-cloud/README.md:104]: Spacing

File: transfer/transfer-05-file-transfer-cloud/README.md [hunk 9]
```diff
@@ -38,6 +38,10 @@ cd transfer/transfer-05-file-transfer-cloud/terraform
 terraform init --upgrade
 terraform apply
 ```
+```windows command prompt
+cd transfer\transfer-05-file-transfer-cloud\terraform 
+'other commands work for all OS user, So look up!'
```
Comment (juliapampus) [inline transfer/transfer-05-file-transfer-cloud/README.md:43]:
```suggestion
# Other commands work for all OS user, so look up!
```

File: transfer/transfer-02-file-transfer-listener/README.md [hunk 10]
```diff
@@ -64,6 +72,12 @@ curl -X GET -H 'X-Api-Key: password' "http://localhost:9192/api/v1/management/co
 curl -X POST -H "Content-Type: application/json" -H "X-Api-Key: password" -d @transfer/transfer-01-file-transfer/filetransfer.json "http://localhost:9192/api/v1/management/transferprocess"
 ```
 
+```Windows command prompt
+curl -X POST -H "Content-Type: application/json" -H "X-Api-Key: password" -d @transfer\transfer-01-file-transfer\contractoffer.json "http://localhost:9192/api/v1/management/contractnegotiations"
+'Second command works for all OS user, So look up!'
```
Comment (juliapampus) [inline transfer/transfer-02-file-transfer-listener/README.md:77]: See comments below

File: transfer/transfer-02-file-transfer-listener/README.md [hunk 11]
```diff
@@ -53,6 +53,14 @@ java -Dedc.fs.config=transfer/transfer-02-file-transfer-listener/file-transfer-l
 java -Dedc.fs.config=transfer/transfer-01-file-transfer/file-transfer-provider/config.properties -jar transfer/transfer-01-file-transfer/file-transfer-provider/build/libs/provider.jar
 ````
 
+```windows command prompt
+.\gradlew transfer:transfer-02-file-transfer-listener:file-transfer-listener-consumer:build
+java -Dedc.fs.config=transfer\transfer-02-file-transfer-listener\file-transfer-listener-consumer\config.properties -jar transfer\transfer-02-file-transfer-listener\file-transfer-listener-consumer\build\libs\consumer.jar
+# in another terminal window:
```
Comment (juliapampus) [inline transfer/transfer-02-file-transfer-listener/README.md:59]: Spacing

**Review by juliapampus — CHANGES_REQUESTED** [review]

File: transfer/transfer-02-file-transfer-listener/README.md (hunk 3 above)
Comment (mukta2022) [reply transfer/transfer-02-file-transfer-listener/README.md]: I push the changes for spacing, capitalization. Fow duplication issue, i make an extra comment to safe user from doing unwanted mistake like , (not follow the 'Mac user' commands which have no change for 'windows user' ).

**Pull request closed without merging**